ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -268,7 +268,11 @@ def install_internal_cluster(self):
         self.do_click(
             locator=self.dep_loc["storage_class_dropdown"], enable_screenshot=True
         )
-        self.do_click(locator=self.dep_loc[self.storage_class], enable_screenshot=True)
+        self.do_click(
```

Comment:
Can we make this please in two separate PRs please? One for fixing the azure deployment issue and second for dom export?

Reply:
ok

Reply:
done